content/realtime/metachannels.textile

```diff
@@ -0,0 +1,111 @@
+
+If @[meta]@ is not specified in the permissions, you are unable to access the metachannels. An example of this would be the following:
+
+minimize. See an example of a capability which is unable to use metachannels
+  ```[json]
+  {
+    "*":["*"]
+  }
+  ```
+
+    Although the above provides all capabilities in all regular channels, without a @[meta]@ permission being explicitly specified you are unable to perform any actions on a @[meta]@ channel.
+
+h2(#subscribing-to-metadata-events). Subscribing to metachannel events
+
+Both the "channel lifecycle":#lifecycle-events and the "occupancy":#occupancy events are published onto a special channel named @[meta]channel.lifecycle@, that can be subscribed to with the "realtime libraries":/realtime. This is often used in conjunction with the "Ably Reactor":/reactor, which allows you to send this data to your "serverless endpoints":/general/events, "queues":/general/queues, and "streams":/general/firehose.
+
+h3(#lifecycle-events). Lifecycle Events
+
+The following events are published as messages on the @[meta]channel.lifecycle@ channel. In all cases, the @data@ member of the message is a "ChannelDetails":#channel-details instance.
+
+- channel.opened := indicates that the channel has been activated globally; that is, it has become active in at least one region, having previously been inactive. The embedded @ChannelDetails.status@ includes "occupancy":#occupancy globally.
+- channel.closed := indicates that the channel has been deactivated globally.
+- channel.region.active := indicates that the channel has been activated in a specific region. The included @ChannelDetails.status@ includes "occupancy":#occupancy in that region.
+- channel.region.inactive := indicates that the channel has been deactivated in a specific region. The included @ChannelDetails.status@ includes "occupancy":#occupancy in that region.
+- channel.occupancy := indicates a change in global "occupancy":#occupancy on the channel. Not all "occupancy":#occupancy changes are sent; there is a minimum interval on a per-channel basis, so if the "occupancy":#occupancy is changing continuously, then only periodic snapshots of occupancy are sent. Further roll-up of events may occur depending on the capacity of the lifecycle channel. Any "occupancy":#occupancy changes in which any of the "occupancy":#occupancy categories changes from 0 to a non-zero count, or vice versa, are always included. This feature is only available on a per-app basis to Ably's "enterprise customers":https://www.ably.io/pricing/enterprise. It is recommended that you use "inband channel occupancy":/realtime/inband-occupancy events instead.
+
+The following code snippet shows how to subscribe to channel lifecycle events:
+
+```[javascript]
+var metaChannel = ably.channels.get("[meta]channel.lifecycle");
```

Comment:
We need to be consistent in whether we use single or double quotes

Reply:
I've changed to single quotes throughout.